react-frontend/src/css/footer.css

```diff
@@ -41,4 +41,18 @@ footer {
   :focus {
     outline: 4px solid #3B99FC;
     outline-offset: 1px;
+  }
+
+  @media only screen and (max-width: 900px) {
+    footer ul li a {
```

Comment:
this is way too generic best use a class name

Reply:
Copy pasted from the design system :p I'll just make a class name for this!

Reply:
make an issue with the design system i'd say :)